dcrwallet.go

```diff
@@ -177,6 +178,14 @@ func run(ctx context.Context) error {
 		w, err := loader.OpenExistingWallet(walletPass)
 		if err != nil {
 			log.Errorf("Failed to open wallet: %v", err)
+			appErr := err.(apperrors.E)
+			if appErr.ErrorCode == apperrors.ErrWrongPassphrase {
```

Comment:
use `apperrors.IsError(err, apperrors.ErrWrongPassphrase)` instead which won't panic if the error is not an `apperrors.E`.